src/core/qgsvectorlayer.h

```diff
@@ -41,6 +41,7 @@
 #include "qgsfeatureiterator.h"
 #include "qgsexpressioncontextgenerator.h"
 #include "qgsexpressioncontextscopegenerator.h"
```

Comment:
Changes left in this file very much look like leftovers.
Sidenote: preferably remove all whitespace changes too.

Reply:
done